pkg/store/bucket_test.go

```diff
@@ -1318,8 +1319,6 @@ func benchBucketSeries(t testutil.TB, skipChunk bool, samplesPerSeries, totalSer
 
 	if !t.IsBenchmark() {
 		if !skipChunk {
-			// Make sure the pool is correctly used. This is expected for 200k numbers.
-			testutil.Equals(t, numOfBlocks, int(st.chunkPool.(*mockedPool).gets.Load()))
```

Comment:
Checking the number of slabs allocated by chunk pool is now untrivial as it depends on the time series.